modules/gapi/test/cpu/gapi_core_tests_cpu.cpp

```diff
@@ -143,16 +128,14 @@ INSTANTIATE_TEST_CASE_P(Cart2PolarCPU, Cart2PolarTest,
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(CV_32FC1),
-/*init output matrices or not*/ testing::Bool(),
                                 Values(CORE_CPU)));
 
 INSTANTIATE_TEST_CASE_P(PhaseCPU, PhaseTest,
                         Combine(Values(CV_32F, CV_32FC3),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
-                                Values(SAME_TYPE),
-                                Values(true),
+                                Values(-1),
                                 Values(CORE_CPU),
                                 testing::Bool()));
```

Comment:
what is this bool btw?

Reply:
whether the calculated angle is in degrees or in radians

Reply:
Ok, thanks. Hard to get it without a comment (not your fault though)

Reply:
added comment